dcc-portal-ui/app/scripts/analysis/js/analysisLauncher.js

```diff
@@ -301,6 +363,65 @@
         wait([result.id], 5, proxyLaunch);
       });
     };
+    
+    _this.demoOncogrid = function () {
+      var donorSetParams = {
+        filters: {
+          donor:{
+            primarySite: {is: ['Liver']},
+            studies: {is: ['PCAWG']}
+          },
+          gene: {
+            curatedSetId: {is: ['GS1']}
+          },
+          mutation: {
+            functionalImpact: {is: ['High']}
+          }
+        },
+        size: 75,
+        type: 'donor',
+        isTransient: true,
+        name: 'Top 75 PCAWG Liver Donors'
+      };
+      
+      var geneSetParams = {
+        filters: {
+          donor:{
+            primarySite: {is: ['Liver']},
+            studies: {is: ['PCAWG']}
+          },
+          gene: {
+            curatedSetId: {is: ['GS1']}
+          },
+          mutation: {
+            functionalImpact: {is: ['High']}
+          }
+        },
+        size: 75,
+        type: 'gene',
+        isTransient: true,
+        name: 'Top 75 CGC Genes for Liver'
+      };
+
+      Page.startWork();
+      SetService.addSet('donor', donorSetParams).then(function (r1) {
+        SetService.addSet('gene', geneSetParams).then(function (r2) {
```

Comment:
These two requests look like they can be fired simultaneously, currently 'gene's will only be added after 'donor' is

```js
$q.all({
  r1: SetService.addSet('donor', donorSetParams),
  r2: SetService.addSet('gene', geneSetParams)
}).then(function (responses) {
  var r1 = responses.r1;
  var r2 = responses.r2;
  //...
});
```